extensions/wikia/Chat2/ChatWidget.class.php

```diff
@@ -71,6 +71,7 @@ public static function getTemplateVars( $fromParserTag ) {
 		$vars = [
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Immediately return this expression instead of assigning it to the temporary variable "$vars". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1488)